charts/zookeeper-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: zookeeper-operator
-version: 0.2.2
-appVersion: 0.2.2
+version: 0.2.5
```

Comment:
Please change to `latest` so we don;t have to keep editing this file all the time.

Reply:
done